modulus/experimental/resdiff/conf/config_generate.yaml

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+hydra:
+  job:
+    chdir: False
+
+
+## Main options
+res_ckpt_filename: "./checkpoints/diffusion.pkl"   
+  # Checkpoint filename for the generative model  
+reg_ckpt_filename: "./checkpoints/regression.pkl"
+  # Checkpoint filename for the mean predictor model
+image_outdir: "image_outdir"
+  # Where to save the output images
+seeds: "0-63"
+  # Random seeds used for generation
+class_idx: null  
+  # Class label. Null is random
+num_steps: 18
+  # Number of sampling steps
+sample_res: "full"
+  # Sampling resolution
+pretext: null
+  # String to include the task
+res_edm: true
+  # If residual-based edm is used
+sampling_method: "stochastic"
+  # Sampling method ["stochastic", "deterministic"]
+
+## Data options
+train_data_path: "s3://sw_climate_fno/nbrenowitz/2023-01-24-cwb-4years.zarr"
```

Comment:
Isn't this path the internal one we have? If possible probably not include this.

Reply:
Sure, will remove this